components/SnippetDetails.js

```diff
@@ -28,10 +31,17 @@ function SnippetDetails() {
       <Link href={link}>Further information</Link>
       <Heading>Tag</Heading>
       <p>{tags}</p>
+
+      <Link href={`/${id}/edit`}>
+        <span aria-label="hidden">✏️ </span> Edit
```

Comment:
```suggestion
        <span role="img" aria-label="hidden">✏️ </span> Edit
```

Reply:
Hi there, thank you for your feedback! We incorporated your suggestion, but weren't 100% sure why the role is important here. From our understanding, the role-attribute is only important for screen-readers. However, by setting the aria-label to "hidden", we told screen-readers to ignore the span-element. So, why is the role-attribute still needed?